src/lib/Behat/Page/ContentUpdateItemPage.php

```diff
@@ -219,9 +220,21 @@ private function getFieldtypeIdentifier(VisibleCSSLocator $fieldLocator, string
         }
 
         $fieldClass = $this->getHTMLPage()->find($fieldLocator)->getAttribute('class');
-        preg_match('/ibexa-field-edit--[ez|ibexa][a-z_]*/', $fieldClass, $matches);
+        $pattern = '/ibexa-field-edit--[ez|ibexa][a-z_]*/';
 
-        return explode('--', $matches[0])[1];
+        preg_match($pattern, $fieldClass, $matches);
+
+        if (empty($matches)) {
+            throw new RuntimeException(sprintf(
+                'Cannot match results for pattern: "%s" and subject: "%s".',
+                $pattern,
+                $fieldClass
+            ));
+        }
+
+        $matchedResults = explode('--', $matches[0]);
```

Comment:
Are we sure that `$matches` is not empty and `$matchedResults` has at least two values?

Reply:
Yeah, the regular expression kinda enforces that on its own.